app/models/ingest_job.rb

```diff
@@ -470,9 +470,16 @@ def generate_success_email_array
 
     case file_type
     when /Matrix/
-      genes = Gene.where(study_id: self.study.id, study_file_id: self.study_file.id).count
-      message << "Gene-level entries created: #{genes}"
-      mixpanel_log_props.merge!({:numGenes => genes})
+      # since genes are not ingested for raw counts matrices, report number of cells ingested
+      if self.study_file.is_raw_counts_file?
+        cells = self.study.expression_matrix_cells(self.study_file).count
```

Comment:
FYI: I *think* these 'self's are unneeded, but I'm not sure.  (and don't mind if they're left in even if they're unneeded).  Rubocop has a really good cop for detecting unecessary 'self' statements, so sometime after this release, we should run it and clean out all the extraneous selfs

Reply:
Agreed, we had said previously that we wanted to do a full pass of `rubocop`.  And you're correct that they aren't needed - I'm just pedantic about `self`.  But it would be good if we were consistent across the codebase.